Unity-Technologies/ScriptableRenderPipeline
# remove all calls to cacheAsBitmap.

### Purpose of this PR
Removing calles of deprecated cacheAsBitmap from VFX. 
---
### Release Notes
Please add any useful notes about the feature/fix that might be helpful for others.

---
### Testing status
**Katana Tests**: First off we need to make sure the Katana SRP tests are green?

**Manual Tests**: 
Checked that the performance where still good with as big graph. Checked that there is no more warnings.


**Automated Tests**: What did you setup? (Add a screenshot or the reference image of the test please)

Launch Katana (Link on master here - update for your branch):
Unneeded no test check ui repaint.

Alternative, launch Yamato (Select your branch):
https://yamato.prd.cds.internal.unity3d.com/jobs/78-ScriptableRenderPipeline

Any test projects to go with this to help reviewers?

---
### Overall Product Risks
**Technical Risk**: None, Low, Medium, High?
Low - removed call that has already no effect.
**Halo Effect**: None, Low, Medium, High?
Low - vfx only.
---
### Comments to reviewers
Notes for the reviewers you have assigned.
